simple_registers/counters/counter16_4/Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile
+
+# defaults
+SIM ?= icarus
+TOPLEVEL_LANG ?= verilog
+
+VERILOG_SOURCES += counter16_4.v
```

Comment:
For the naming of counter with multiple clocks, we need to also define it in the naming convention.
I propose to add a new keyword like ``<int>clk``

```
   counter[down]<size>_<num_clocks>clk_[async|sync]_[setp|resetp|setn|resetn]
```

Let me know what you think.